src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -130,6 +135,8 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
                     $attributeMetadata->setSerializedName($annotation->getSerializedName());
                 } elseif ($annotation instanceof Ignore) {
                     $attributeMetadata->setIgnore(true);
+                } elseif ($annotation instanceof Context) {
```

Comment:
Shouldn't you throw a `MappingException` if it's not an accessor or mutator?

Reply:
Good catch, thank you